models/modelTopics.js

```diff
@@ -2,13 +2,9 @@ const db = require("../db/connection");
 
 function selectTopicData() {
   return db.query(`SELECT * FROM topics`).then((result) => {
-    const formattedData = result.rows.map((topic) => {
-        return {
-            slug: topic.slug,
-            description: topic.description
-        }
-    })
-    return formattedData
+    return {
+        topics: result
```

Comment:
Out of convention, I would return `result` from the model and then put into an object like `{topics : result}` in the controller